recipes/_update_shcluster.rb

```diff
@@ -0,0 +1,28 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: _update_shcluster
+#
+# Recipe determines whether to add/remove this SH member to/from SH Cluster.
+
+password_file = File.join node['splunk']['external_config_directory'], 'password'
+search_heads = CernerSplunk.my_cluster_data(node)['sh_cluster']
+
+fail 'Search Heads are not configured for sh clustering in the cluster databag' if search_heads.nil? || search_heads.empty?
+
+notifies_action = node['splunk']['config']['sh_member'].nil? ? :nothing : :stop
+if node['splunk']['config']['sh_member'] == 'add'
```

Comment:
nit: We use `node['splunk']['config']['sh_member']` frequently enough that I think it should be assigned a variable.